.github/actions/run_unit_tests/action.yml

```diff
@@ -30,12 +29,6 @@ runs:
         key: ${{ inputs.ruby-version }}
         map: |
           {
-            "2.4.10": {
-              "rails": "norails,rails52,rails51,rails50,rails42,rails41,rails40"
-            },
-            "2.5.9": {
-              "rails": "norails,rails61,rails60,rails52,rails51,rails50,rails42"
-            },
```

Comment:
Does this mean we need to mention we're dropping support for 4.1 / 4.0? Or just leave it untested?